pymfe/landmarking.py

```diff
@@ -910,7 +910,7 @@ def ft_one_nn(
 
         model = sklearn.neighbors.KNeighborsClassifier(
             n_neighbors=1,
-            algorithm="auto",
+            algorithm="kd_tree",
```

Comment:
In fact, "auto" is not a good option, but I also can't say whether "kd-tree" or "ball-tree" is better. In the end, it is better to impose some bias for everything than to use several (like "auto"). Thank you for observing this.

Reply:
In that case, my intention is to just keep the unit tests working. I don't know what changed in later versions of the Scikit-learn, but I'm sure that the heuristic used to select the algorithm (either "kd-tree" or "ball-tree") has be modified and, therefore, the previous unit tests results were not matching anymore.

If you find better to use another algorithm, or even "auto" in this case, we can change the unit tests for that method if necessary.